build.gradle

```diff
@@ -18,6 +18,7 @@
 buildscript {
     repositories {
         jcenter()
+        mavenCentral()
```

Comment:
Fix 
![image](https://user-images.githubusercontent.com/11749867/182512609-f2a9ca10-c855-402c-9e4e-d892fdce3d5d.png)
`scalafmt-core` is deprecated in jcenter, should use `mavenCentral()`
And need to keep jcenter() due to exist some lib needs this